CODEOWNERS

```diff
@@ -0,0 +1,15 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# More details are here: https://help.github.com/articles/about-codeowners/
+
+# The '*' pattern is global owners.
+
+# Order is important. The last matching pattern has the most precedence.
+# The folders are ordered as follows:
+
+# In each subsection folders are ordered first by depth, then alphabetically.
+# This should make it easy to add new rules without breaking existing ones.
+
+# Global rule:
+*       @aaronjpitty @almarcotte @danmrichards @lwaddicor @soulsaengs @stevenh
```

Comment:
How is this list generated?

Reply:
Not generated, implemented by hand right now.

Longer term, a better approach would be to create a `@go-svrquery-maintainers` group and have that be the codeowner. Then we can add/remove folks without needing to change the file.

Reply:
Actually, I'll just do that now instead.